test/pytest/test_softmax.py

```diff
@@ -10,50 +10,47 @@
 test_root_path = Path(__file__).parent
 
 
-def flat_distribution(shape):
-    return np.random.rand(*shape)
-
-
-def high_accuracy_distribution(shape):
-    '''Start with a flat distribution, then pick a random member of each row to amplify'''
-    x = np.random.rand(*shape)
-    imax = np.random.randint(0, shape[1], size=shape[0])
-    x[:, imax] *= 10
-    return x
+def normal_dist(shape):
+    return np.clip(np.random.normal(0, 8, shape), -32, 31)
```

Comment:
Why are we getting rid of existing distributions?

Reply:
The existing distribution generation does not feel necessary, and I don't see a good reason to run tests twice on uniform and uniform+manual outliers. A gaussian will also give variables of many scales, giving some outliers by default.

Reply:
In this case I shall purge this part, should an equivalent test already exists somewhere.